CodingChallenges/CC_029_SmartRockets/Processing/CC_029_SmartRockets/Dna.pde

```diff
@@ -0,0 +1,50 @@
+// Daniel Shiffman
+// http://codingtra.in
+// http://patreon.com/codingtrain
+// Code for: https://youtu.be/bGz7mv2vD6g
+// Processing transcription: Chuck England
+
+import java.util.*;
+
+int lifespan = 400;
+
+class DNA {
+  List<PVector> genes;
```

Comment:
Would a plain array be simpler to follow here given the lifespan is fixed?

Reply:
I’ll take a look again. But I was thinking about the previous comments when I did it, and I think the genes were dynamic.

Reply:
Ok, you were right, doh! I guess I saw the .add() I used and thought it was dynamic. Updated.

Reply:
![image](https://user-images.githubusercontent.com/1686727/50668146-200e3080-0f72-11e9-9cac-9724c38f120b.png)